doc/sphinxman/source/simint.rst

```diff
@@ -0,0 +1,141 @@
+.. sectionauthor:: Lori A. Burns
+
+.. *Module:* :ref:`Keywords <apdx:efp>`, :ref:`PSI Variables <apdx:efp_psivar>`, :source:`LIBEFP <src/lib/libefp_solver>`
+
+.. image:: https://img.shields.io/badge/home-simint-5077AB.svg
+   :target: http://www.bennyp.org/research/simint/
+
+.. raw:: html
+
+   <br>
+
+.. image:: https://img.shields.io/badge/docs-latest-5077AB.svg
+   :target: http://www.bennyp.org/research/simint/
+
+These are the vectorized implementation of the Obara-Saika (OS) method of
+calculating electron repulsion integrals developed by B. Pritchard and
+interfaced into libmints. Enabling simint and adding ``set
+integral_package simint`` (do this in ``~/.psi4rc`` for universal effect)
+runs libderiv from libint for derivative integrals and simint for
+non-derivative integrals. Note that present AM maximum is ``$$(gg|gg)$$``
+
+Installation
+~~~~~~~~~~
+
+**Binary**
+
+* .. image:: https://anaconda.org/psi4/simint/badges/version.svg
+     :target: https://anaconda.org/psi4/simint
+
+* SIMINT is available as a conda package for Linux and macOS.
+
+.. * If using the |PSIfour| binary, simint has already been installed alongside.
+
+* If using |PSIfour| built from source, and anaconda or miniconda has
+  already been installed (instructions at :ref:`sec:quickconda`),
+  simint can be obtained through ``conda install simint``.
+  Then enable it as a feature with :makevar:`ENABLE_simint`,
+  hint its location with :makevar:`CMAKE_PREFIX_PATH`,
+  and rebuild |PSIfour| to detect simint and activate dependent code.
+
+* To remove a conda installation, ``conda remove simint``.
+
+**Source**
+
+* .. image:: https://img.shields.io/github/tag/psi4/simint.svg?maxAge=2592000
+..     :target: https://github.com/psi4/simint TODO BPP
+
+* If using |PSIfour| built from source and you want simint built from
+  from source also,
+  enable it as a feature with :makevar:`ENABLE_simint`,
+  and let the build system fetch and build it and activate dependent code.
+
+
+.. _`cmake:simint`:
+
+How to configure simint for building Psi4
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Role and Dependencies**
+
+* Role |w---w| In |PSIfour|, simint is a library that provides alternate
+  integrals.
+
+* Downstream Dependencies |w---w| |PSIfour| (\ |dr| optional) simint
+
+* Upstream Dependencies |w---w| simint |dr| None
+
+**CMake Variables**
+
+* :makevar:`ENABLE_simint` |w---w| CMake variable toggling whether |PSIfour| builds with simint
+* :makevar:`CMAKE_PREFIX_PATH` |w---w| CMake list variable to specify where pre-built dependencies can be found. For simint, set to an installation directory containing ``include/simint/simint.h``
+* :makevar:`simint_DIR` |w---w| CMake variable to specify where pre-built simint can be found. Set to installation directory containing ``share/cmake/simint/simintConfig.cmake``
+* :makevar:`CMAKE_DISABLE_FIND_PACKAGE_simint` |w---w| CMake variable to force internal build of simint instead of detecting pre-built
```

Comment:
@loriab , I'm not suggesting that we should auto-detect the vectorization scheme (although it can be done using `#ifdef __SSE__`,`#ifdef __SSE2__`,`#ifdef __SSE3__`,`#ifdef __AVX__` and `#ifdef __MIC__` at compile time and a call to `cpuid` at run time).  One of the potential pitfalls of that would be compiling on one node, then running on different architectures, which would have to be accounted for using the above mechanism.  I think the easiest thing to do is to set the default to AVX, as @dgasmith suggested, and put a warning in the documentation and/or CMakeLists.txt that this may not be appropriate.  I can provide code snippets for testing cpu features if you like, but I advocate the simple approach.